src/deluge/playback/mode/session.cpp

```diff
@@ -1700,7 +1708,8 @@ void Session::armClipToStartOrSoloUsingQuantization(Clip* thisClip, bool doLateS
 		// If late start...
 		if (doLateStart) {
 
-			if (thisClip->armState != ArmState::OFF) { // In case also already armed
+			if (thisClip->armState != ArmState::OFF && thisClip->launchStyle != LaunchStyle::FILL
```

Comment:
Check for default instead

Reply:
Same as before

Reply:
I think this should be if launchstyle == default as well. Reason being that positive conditions are simpler to add to later

Reply:
Done